staging/src/k8s.io/apiserver/pkg/server/config_test.go

```diff
@@ -32,6 +32,56 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+func TestAuthorizeClientBearerTokenNoops(t *testing.T) {
+	// All of these should do nothing (not panic, no side-effects)
+	cfgGens := []func() *rest.Config{
+		func() *rest.Config { return nil },
+		func() *rest.Config { return &rest.Config{} },
+		func() *rest.Config { return &rest.Config{BearerToken: "mu"} },
+	}
+	authcGens := []func() *AuthenticationInfo{
```

Comment:
authnGens?

Reply:
"auth*n*" matches both "authentication" and "authorization"; "auth*c*" matches only "authentication"